memcached-operator/deploy/olm-catalog/memcached-operator/0.0.1/memcached-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,169 @@
+            selector:
+              matchLabels:
+                name: memcached-operator
+            strategy: {}
+            template:
+              metadata:
+                labels:
+                  name: memcached-operator
+              spec:
+                containers:
+                  - command:
+                      - memcached-operator
+                    env:
+                      - name: WATCH_NAMESPACE
+                        valueFrom:
+                          fieldRef:
+                            fieldPath: metadata.annotations['olm.targetNamespaces']
+                      - name: POD_NAME
+                        valueFrom:
+                          fieldRef:
+                            fieldPath: metadata.name
+                      - name: OPERATOR_NAME
+                        value: memcached-operator
+                    image: quay.io/example/memcached-operator:v0.0.1
+                    imagePullPolicy: Never
+                    name: memcached-operator
+                    resources: {}
+                serviceAccountName: memcached-operator
+      permissions:
+        - rules:
+            - apiGroups:
+                - ""
+              resources:
+                - pods
+                - services
+                - endpoints
+                - persistentvolumeclaims
+                - events
+                - configmaps
+                - secrets
+              verbs:
+                - '*'
+            - apiGroups:
+                - ""
+              resources:
+                - namespaces
+              verbs:
+                - get
+            - apiGroups:
+                - apps
+              resources:
+                - deployments
+                - daemonsets
+                - replicasets
+                - statefulsets
+              verbs:
+                - '*'
+            - apiGroups:
+                - monitoring.coreos.com
+              resources:
+                - servicemonitors
+              verbs:
+                - get
+                - create
+            - apiGroups:
+                - apps
+              resourceNames:
+                - memcached-operator
+              resources:
+                - deployments/finalizers
+              verbs:
+                - update
+            - apiGroups:
+                - cache.example.com
+              resources:
+                - '*'
+                - memcachedrs
+              verbs:
+                - '*'
+          serviceAccountName: memcached-operator
+    strategy: deployment
+  installModes:
+    - supported: true
+      type: OwnNamespace
+    - supported: true
+      type: SingleNamespace
+    - supported: false
+      type: MultiNamespace
+    - supported: true
+      type: AllNamespaces
+  keywords:
+    - memcached
+    - app
+  maintainers:
+    - email: corp@example.com
+      name: Some Corp
+  maturity: alpha
+  provider:
+    name: Example
+    url: www.example.com
+  version: 0.0.1
```

Comment:
Is there a reason we deleted the existing CSVs and created a new one at a lower semver version? I would think we should just create a new CSV at `v0.10.1` or `v0.11.0` depending on whether there are breaking changes.

Reply:
- The previous files were just generated with operator-sdk but was not updated then they never actually worked. What is the point to keep them? But if you wish, I can come back them to the project.
- Regards the version, see that we are using the tag 0.0.1 so to keep aligned with it. Note that the tags used before shows that were matching with the SDK releases and not the breaking changes into it. 

IMHO, we could apply the fix since now this CSV will work and start to upgrade/change it based on the changes in the project. WDYT? Keep it as it is now in this PR or came back and just add the 0.11.0 on it so?

Reply:
The current project structure suggests that each release of the SDK has a corresponding version of memcached-operator and CSV. That CSV should therefore reference:

1. The new memcached-operator image, ex. `quay.io/example/memcached-operator:v0.y.0`
1. The previous memcached-operator version in `spec.replaces`, ex. `v0.x.0`
1. The new memcached-operator version in `spec.version`, ex. `v0.y.0`

where x is the old and y is the new minor version. The CSV's should be tested to make sure they are not broken on each release of this repo.

We might want to consider building and pushing example memcached-operator images to quay.io to maintain reproducibility.

Reply:
After a chat with @estroz was decided came them back and keep the new one as 0.11.0